trustmanager/keyfilestore.go

```diff
@@ -51,34 +141,64 @@ func (s *KeyFileStore) Name() string {
 }
 
 // AddKey stores the contents of a PEM-encoded private key as a PEM block
-func (s *KeyFileStore) AddKey(name, role string, privKey data.PrivateKey) error {
+func (s *KeyFileStore) AddKey(privKey data.PrivateKey, keyInfo KeyInfo) error {
```

Comment:
Can we put KeyInfo first? I guess my thing with these kind of storage related functions is I really like the arg order to be (identifier, data)